### PR TITLE
Bump the Swift version to 5.7.3

### PR DESCRIPTION
**Pull request by shahmishal**
https://github.com/apple/swift/pull/63038

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci smoke test

**Review by MaxDesiatov — APPROVED** [review]
Are we skipping 5.7.2 here because we forgot to make this patch version bump in SwiftPM?

**Review by tomerd — APPROVED** [review]

**Comment by shahmishal** [comment]
> Are we skipping 5.7.2 here because we forgot to make this patch version bump in SwiftPM?

5.7.2 is already released

File: Sources/Basics/SwiftVersion.swift [hunk 1]
```diff
@@ -54,7 +54,7 @@ public struct SwiftVersion {
 extension SwiftVersion {
     /// The current version of the package manager.
     public static let current = SwiftVersion(
-        version: (5, 7, 1),
```
Comment (Sulxl) [inline Sources/Basics/SwiftVersion.swift:57]: 3

**Review by Sulxl — COMMENTED** [review]
Active

**Review by same2508 — COMMENTED** [review]
Sources/Basics/SwiftVersion.swift

**Pull request merged**